packages/slo-stat-bot/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "slo-stat-label",
+  "version": "0.0.1",
+  "description": "Labels SLO status on Issues and PRs",
+  "private": true,
+  "author": "Google LLC.",
+  "license": "Apache-2.0",
+  "repository": "https://github.com/googleapis/repo-automation-bots.git",
+  "homepage": "https://github.com/googleapis/repo-automation-bots",
+  "bugs": "https://github.com/googleapis/repo-automation-bots/issues",
+  "main": "build/src/app.js",
+  "keywords": [
+    "slo",
+    "probot",
+    "github",
+    "probot-app"
+  ],
+  "files": [
+    "build/src"
+  ],
+  "scripts": {
+    "compile": "tsc -p .",
+    "start": "probot run ./build/src/slo-stat-label.js",
+    "start:local": "node ./build/src/local.js",
+    "prepare": "npm run compile",
+    "pretest": "npm run compile",
+    "test": "cross-env LOG_LEVEL=fatal c8 mocha --exit build/test",
+    "test:snap": "SNAPSHOT_UPDATE=1 npm test",
+    "fix": "gts fix",
+    "lint": "gts check"
+  },
+  "dependencies": {
+    "@octokit/rest": "^16.41.2",
+    "@types/chai-spies": "^1.0.1",
+    "assert": "^2.0.0",
+    "chai-spies": "^1.0.0",
```

Comment:
I took a closer look, and it doesn't look like you're using `chai` at all!  Maybe remove these dependencies from `package.json`?